gateway/api/tasks/providers.py

```diff
@@ -20,25 +20,31 @@ def assign_admin_group():
     """
     try:
         providers_configuration = json.loads(settings.PROVIDERS_CONFIGURATION)
-    except json.JSONDecodeError as e:
-        logger.error("Assign admin group JSON malformed: %s", e)
+    except json.JSONDecodeError:
+        logger.error(
+            "Assign admin group JSON malformed in settings.PROVIDERS_CONFIGURATION"
+        )
         return
-    except Exception as e:  # pylint: disable=broad-exception-caught
-        logger.error("Assign admin group unexpected error: %s", e)
+    except Exception:  # pylint: disable=broad-exception-caught
```

Comment:
did the log of the error get flagged on the security check? otherwise might be nice to keep for debugging

Reply:
Yes, the static scan complains about this.